test/Utility.test.ts

```diff
@@ -48,3 +48,10 @@ describe("#randomPointWithinCircle", () => {
     }
   });
 });
+
+describe("#getRandomInt", () => {
+  test("should generate a random integer less than the specified max", () => {
+    expect(Utility.getRandomInt(3)).toBeLessThan(3);
```

Comment:
this is a good start... 
For this test, we can do a for-loop which tries to generate the number lets say 20 times... 

```
for(let i=0; I < 20; i++){
expect(Utility.getRandomInt(3)).toBeLessThan(3);
}
```
This is a more solid test because we know that it will try to generate a number between 0 and 3 - 20 times. If all the 20 times pass, we are guaranteed that the max is being respected.